src/analysis/modals/CloudEnvironmentModal.ts

```diff
@@ -460,14 +460,16 @@ export const CloudEnvironmentModal = ({
         () => {
           return {
             ...baseProps,
-            href:
-              app &&
-              (cloudProvider === cloudProviderTypes.AZURE
-                ? app?.proxyUrls['cbas-ui']
-                : app?.proxyUrls['cromwell-service']),
+            href: app && (cloudProvider === cloudProviderTypes.GCP ? app?.proxyUrls['cromwell-service'] : ''),
             onClick: () => {
               onDismiss();
               Metrics(signal).captureEvent(Events.applicationLaunch, { app: appTools.CROMWELL.label });
+              if (
+                cloudProvider === cloudProviderTypes.AZURE &&
+                Nav.getCurrentRoute().name !== 'workspace-workflows-app'
```

Comment:
Do we also want to check that Workflows tab featured preview is enabled? Otherwise I think it would redirect the user to the Workflows tab landing page without the tab actually being enabled

Reply:
yes, you're right!